src/main/java/seedu/clinic/logic/commands/EditCommand.java

```diff
@@ -85,11 +85,31 @@
     public static final String MESSAGE_WAREHOUSE_UNCHANGED = "The edited field will results in no change to "
             + "warehouse selected. Please check your arguments again and re-enter your edit command.";
 
+    private static final String LOG_MESSAGE_RECEIVE_EDIT_SUPPLIER = "Received instructions to edit supplier";
+    private static final String LOG_MESSAGE_RECEIVE_EDIT_WAREHOUSE = "Received instructions to edit warehouse";
+    private static final String LOG_MESSAGE_SUPPLIER_RETRIEVED = "Supplier to edit is retrieved from supplier list.";
+    private static final String LOG_MESSAGE_WAREHOUSE_RETRIEVED = "Warehouse to edit is retrieved from warehouse list.";
+    private static final String LOG_MESSAGE_SUPPLIER_EDITED = "Supplier with edited information has been created.";
+    private static final String LOG_MESSAGE_WAREHOUSE_EDITED = "Warehouse with edited information has been created.";
+    private static final String LOG_MESSAGE_SUPPLIER_REPLACED_IN_MODEL = "Replaced supplier in supplier list.";
+    private static final String LOG_MESSAGE_WAREHOUSE_REPLACED_IN_MODEL = "Replaced warehouse in warehouse list.";
+    private static final String LOG_SUPPLIER_UPDATED_IN_UI = "Updated supplier in UI.";
+    private static final String LOG_WAREHOUSE_UPDATED_IN_UI = "Updated warehouse in UI.";
+
+    private static final String INVALID_EDIT_SUPPLIER_DESCRIPTOR_ASSERTION = "editDescriptor supplied should be "
+            + "of EditWarehouseDescriptor type here.";
```

Comment:
Should this be EditSupplierDescriptor type?

Reply:
Nice spot! Thanks!